examples/cloudquotas/v1beta1/quotapreference.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: cloudquotas.gcp.upbound.io/v1beta1
+kind: QuotaPreference
+metadata:
+  annotations:
+    meta.upbound.io/example-id: cloudquotas/v1beta1/quotapreference
+  labels:
+    testing.upbound.io/example-name: preference
+  name: preference
+spec:
+  forProvider:
+    contactEmail: testuser@gmail.com
+    dimensions:
+      region: us-east1
+    parent: projects/my-project-name
+    quotaConfig:
```

Comment:
The code gen was producing an array here and that was leading to test failures. I inspected the generated code and the `QuotaConfigParameters` property is **_not_** a slice in the generated code.

Seems like something to nip in the bud lest it keep wanted to clobber my bespoke tuning?

Reply:
This is a missing/not handled issue in our code generation pipeline. I opened an [issue](https://github.com/crossplane/upjet/issues/483) for this on Upjet after you came across this, feel free to leave a comment there. 

However, from what I see here, you have removed the generated example. Changing the generated examples is not recommended and will cause errors in the CI pipeline. After running make generate, please leave the generated example as is and make the corrections in the copied one.